client/me/sidebar/index.jsx

```diff
@@ -114,6 +114,15 @@ class MeSidebar extends Component {
 							preloadSectionName="security"
 						/>
 
+						<SidebarItem
+							selected={ itemLinkMatches( '/domains', path ) }
+							link="/me/domains"
+							label={ translate( 'Domains' ) }
+							materialIcon="settings"
```

Comment:
We've been using this globe item everywhere to represent all domains.

```suggestion
							materialIcon="language"
```